Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -36,7 +37,7 @@ public void RoslynPlugin_GenerateForValidAnalyzer_Succeeds()
             string packageId = "analyzer1.pkgid1";
             string localNuGetDir = TestUtils.CreateTestDirectory(this.TestContext, ".localNuGet");
             IPackageManager localNuGetStore = CreatePackageManager(localNuGetDir);
-            AddPackage(localNuGetStore, packageId, "1.0.2", analyzer.GetType().Assembly.Location);
+            IPackage analyzerPkg =  AddPackage(localNuGetStore, packageId, "1.0.2", analyzer.GetType().Assembly.Location);
```

Comment:
Need to return a reference to the package so we can check the plugin manifest against it.